README.md

```diff
@@ -49,6 +49,9 @@ Type: `object`
 
 Target options, *see [wiki](https://github.com/kong/httpsnippet/wiki/Targets) for details*
 
+*Note: This fork has added "useObjectBody" to the node target, a boolean flag that includes an object literal wrapped in JSON.stringify, as opposed to a string object body.*
```

Comment:
Can you move this to the bottom of the readme? We've got a block down there for the differences between upstream and our fork.